File: demos/jans-chip/android/app/src/main/java/io/jans/chip/modal/appIntegrity/DeviceIntegrity.java
package io.jans.chip.modal.appIntegrity;

import java.util.List;

public class DeviceIntegrity{
    private List<String> deviceRecognitionVerdict;

    public List<String> getDeviceRecognitionVerdict() {
        return deviceRecognitionVerdict;
    }

    public void setDeviceRecognitionVerdict(List<String> deviceRecognitionVerdict) {
        this.deviceRecognitionVerdict = deviceRecognitionVerdict;
    }

    public String commasSeparatedString(){
        if(this.deviceRecognitionVerdict != null) {
            return String.join(",", this.deviceRecognitionVerdict);
        }
        return null;
    }
}

File: demos/jans-chip/android/app/src/main/java/io/jans/chip/model/appIntegrity/DeviceIntegrity.kt
package io.jans.chip.model.appIntegrity

data class DeviceIntegrity (
    var deviceRecognitionVerdict: List<String>? = null
) {
    fun commasSeparatedString(): String? {
        return if (deviceRecognitionVerdict != null) {
            java.lang.String.join(",", deviceRecognitionVerdict)
        } else null
    }
}